fetch-node.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var wrapFetchForNode = function (fetch) {
+    // Support schemaless URIs on the server for parity with the browser.
+    // https://github.com/matthew-andrews/isomorphic-fetch/pull/10
+    return function (url, options) {
+        if (/^\/\//.test(url)) {
```

Comment:
This seems odd to me (unless I'm missing something). I'm not sure it makes sense to have a schemaless request from a server. That said, I see no harm in what probably won't get used and I gather that they added this for a reason. It might be more safely implemented as:

```javascript
if (url.slice(0, 2)) === '//') { // ...
```